README.md

```diff
@@ -1,2 +1,46 @@
 # Zalando Open Source Team 
 
+The repository and issue tracker of the Zalando Open Source Team (Contact: opensource@zalando.de). Includes the team website and issue tracker with the tasks we are currently working on. 
+
+opensource.zalando.net
+
+
+## The purpose of the open source team
+The Zalando Open Source Team is responsible for nurturing and growing open source and open practices at Zalando. 
+We work across departments as internal consultants to remove barriers for contributing, producing and consuming open
+source code. 
+
+We advice and assist on:
+
+- Creating and nurturing open source communities around Zalando Open Source projects
+- Align community interests with Zalando business interests
+- Efficient and compliant adobtion of open source code by Zalando teams
+- Recruiting from open source communities 
+- Best practices for applying open source to internal projects
+
+We are also here to demystify and simply answer questions, so no matter if you are working at Zalando or not, get in touch: opensource@zalando.de
```

Comment:
😮 demystify 🎊

Reply:
(verb) - make (a difficult subject) clearer and easier to understand  :)